modules/loader/src/main/java/com/opengamma/strata/loader/csv/TradeCsvLoader.java

```diff
@@ -424,6 +433,11 @@ private TradeInfo parseTradeInfo(CsvRow row) {
     TradeInfoBuilder infoBuilder = TradeInfo.builder();
     String scheme = row.findField(ID_SCHEME_FIELD).orElse(DEFAULT_TRADE_SCHEME);
     row.findValue(ID_FIELD).ifPresent(id -> infoBuilder.id(StandardId.of(scheme, id)));
+    String schemeCpty = row.findField(CTPY_SCHEME_FIELD).orElse(DEFAULT_CTPY_SCHEME);
```

Comment:
Should be `findValue()` not `findField()`. This will then default empty schemes.